apps/crn-server/src/data-providers/reminders.data-provider.ts

```diff
@@ -500,7 +500,7 @@ const getEventMaterialsRemindersFromQuery = (
 
 const getSortDate = (reminder: ReminderDataObject): DateTime => {
   if (reminder.entity === 'Research Output') {
-    return DateTime.fromISO(reminder.data.addedDate);
+    return DateTime.fromISO(reminder.data.addedDate || '');
```

Comment:
this one 2

Reply:
I'm not sure about how reminders are handling draft research outputs, but it might be a good idea to separate the Published ROs and Draft ROs as from now based on having or not an addedDate.